pkg/apis/eventbus/v1alpha1/kafka_eventbus.go

```diff
@@ -0,0 +1,43 @@
+package v1alpha1
+
+import (
+	apicommon "github.com/argoproj/argo-events/pkg/apis/common"
+)
+
+// KafkaBus holds the KafkaBus EventBus information
+type KafkaBus struct {
+	// Exotic holds an exotic Kafka config
+	Exotic *KafkaConfig `json:"exotic,omitempty" protobuf:"bytes,1,opt,name=exotic"`
+}
+
+type KafkaConfig struct {
+	// URL to kafka cluster, multiple URLs separated by comma
+	URL string `json:"url,omitempty" protobuf:"bytes,1,opt,name=url"`
+	// Topic name, defaults to namespace_name.eventbus_name
+	// +optional
+	Topic string `json:"topic,omitempty" protobuf:"bytes,2,opt,name=topic"`
+	// Kafka version, sarama defaults to the oldest supported stable version
+	// +optional
+	Version string `json:"version,omitempty" protobuf:"bytes,3,opt,name=version"`
+	// TLS configuration for the kafka client.
+	// +optional
+	TLS *apicommon.TLSConfig `json:"tls,omitempty" protobuf:"bytes,4,opt,name=tls"`
+	// SASL configuration for the kafka client
+	// +optional
+	SASL *apicommon.SASLConfig `json:"sasl,omitempty" protobuf:"bytes,5,opt,name=sasl"`
+	// Consumer group for kafka client
+	// +optional
+	ConsumerGroup *KafkaConsumerGroup `json:"consumerGroup,omitempty" protobuf:"bytes,6,opt,name=consumerGroup"`
+}
```

Comment:
So, basically if user specifies no ConsumerGroup and they have a Sensor with multiple replicas then they will each receive copies of the message, right? Is the expectation that if they don't specify one then they should just be using one replica?

Reply:
Just for my own education, is there a reason they would want no ConsumerGroup?

Reply:
I see now that I think you set ConsumerGroup if they don't define anything, right?

Reply:
Exactly, we need a consumer group. Kafka does have a way to consume from a single partition (and this is an option with the Kafka EventSource) but for the Kafka EventBus we need to use a ConsumerGroup. You can either set it explicitly or use the [default](https://github.com/argoproj/argo-events/pull/2502/files#diff-b778225ecfb035ee9a44077f3f987fd20bc3381f1786485f86639419b0c9196fR48-R53).

Reply:
Yes, let me start by adding a blurb in the generate docs 👍